hax/treemakers/se_treemaker.py

```diff
@@ -0,0 +1,64 @@
+import hax
```

Comment:
Add a docstring above here to explain what is in this module.  Also, the filename should be more explicit: single_electron.py